doc/cephfs/fs-volumes.rst

```diff
@@ -1422,18 +1422,18 @@ set with this id was present in the database
 
 .. _disabling-volumes-plugin:
 
-Disabling Volumes Plugin
-------------------------
+Disabling the Volumes Plugin
```

Comment:
separate pr as well

Reply:
I have moved this into a separate PR here: https://github.com/ceph/ceph/pull/60465.